manifests/prometheus/scripts/generate-manifest.sh

```diff
@@ -23,9 +23,7 @@ done
 
 # shellcheck disable=SC2086
 bosh interpolate \
-  --var-errs \
-  --vars-store "${VARS_STORE}" \
   ${varsfile_args} \
   ${opsfile_args} \
   ${alerts_opsfile_args} \
-  "${PROM_BOSHRELEASE_DIR}/manifests/prometheus.yml"
+  "${PROM_BOSHRELEASE_DIR}/manifests/prometheus.yml" $@
```

Comment:
Shellcheck is complaining about this.

I'm not sure I like this pattern where more and more of the options to `bosh interpolate` get moved outside the script. When we were using spruce, we ended up with the manifest generation script being a very small wrapper around spruce with all the important details being out in the pipeline. This meant you had to verify manually that the tests were calling it in the same way as the pipeline...

Can we instead add a feature-flag to control the vars-store behaviour (eg if the `SKIP_VARS_STORE` env var is set, it skips the corresponding flags).

Reply:
Sure, I can add the flag :+1: 

I shall do the same for the `cf-manifest.yml`

Reply:
@alext have a look, we do the vars-store if the variable `VARS_STORE` is set

Reply:
If you are OK, I can do the same in the cf-manifest stuff

Reply:
Yeah, that makes sense. I like this approach, although we don't need the `$@` on the last line any more...